EIPS/eip-7742.md

```diff
@@ -33,67 +33,95 @@ This EIP also changes how the EL sources the current blob target value for two r
+| `MIN_BASE_FEE_PER_BLOB_GAS_EIP_7742` | `2**25` ||
+| `TARGET_BLOBS_PER_BLOCK_EIP_7742` | `4` | increase the target blobs to `4`|
+| `OLD_TARGET_BLOBS_PER_BLOCK` | `3` | blob target corresponding to [EIP-4844](./eip-4844.md) |
+
+### Block processing
 
 Upon activation of this EIP, execution clients **MUST** extend the header schema with an
-additional 64-bit field: the `target_blobs_per_block`. This value is set to the current target blob count. The Engine API
-is modified along with this EIP to provide the `target_blobs_per_block` with each payload and implementations can use this
-value to correctly set the block header field.
+additional 64-bit field: the `target_blobs_per_block`. This values is set to the one provided by the CL during block production.
 
-Validity of the `target_blobs_per_block` is guaranteed from the consensus layer, much like how withdrawals are handled.
+Any reference to `TARGET_BLOB_GAS_PER_BLOCK` from [EIP-4844](./eip-4844.md) (for e.g. for `excess_blob_gas` calculations) can be derived by taking the `target_blobs_per_block` from the EL block header and multiplying by `GAS_PER_BLOB`.
 
-When verifying a block, execution clients **MUST** ensure the target blob count in the block header matches the one
-provided by the consensus client.
+Validity of these values is guaranteed from the consensus layer, much like how withdrawals are handled. Hence when verifying a block, execution clients **MUST** ensure the target blobs per block in the block header matches the one provided by the consensus client.
 
-For a genesis block with no existing parent, the value should be set according to the agreed specification for the
-target blob count given by that genesis block's protocol rule set.
+Furthermore `get_base_fee_per_blob_gas` as specified by [EIP-4844](./eip-4844.md) is as modified below:
 
-### Block processing
 
-Upon activating this EIP (i.e. before processing any transactions),
-the verification of the blob maximum as given in EIP-4844 can be skipped. Concretely, this means any logic relating
-to `MAX_BLOB_GAS_PER_BLOCK` as given in EIP-4844 can be deprecated.
-Additionally, any reference to `TARGET_BLOB_GAS_PER_BLOCK` from EIP-4844 can be derived by taking the `target_blobs_per_block` from the CL and multiplying by `GAS_PER_BLOB` as given in EIP-4844.
+```python
+def calc_excess_blob_gas(parent: Header) -> int:
+  # normalize parent's excess blob gas if this block was fork block
+  if(parent.timestamp < FORK_TIMESTAMP)
+    normalized_parent_excess_blob_gas = parent.excess_blob_gas * EXCESS_BLOB_GAS_NORMALIZATION_TARGET_BLOBS // OLD_TARGET_BLOBS_PER_BLOCK
+    target_blobs_per_block = OLD_TARGET_BLOBS_PER_BLOCK
+  else
+    normalized_parent_excess_blob_gas = parent.excess_blob_gas
+    target_blobs_per_block = parent.target_blobs_per_block
+
+  return (normalized_parent_excess_blob_gas + (parent.blob_gas_used - target_blob_gas) * EXCESS_BLOB_GAS_NORMALIZATION_TARGET_BLOBS // target_blobs_per_block)
```

Comment:
I believe this can be simplified to 
`return (normalized_parent_excess_blob_gas + (parent.blob_gas_used - target_blob_gas) // target_blob_gas)`

With:
`NEW_BLOB_BASE_FEE_UPDATE_FRACTION_NORMALIZED = BLOB_BASE_FEE_UPDATE_FRACTION_NORMALIZED /128/GAS_PER_BLOB`

Then there is no need for `EXCESS_BLOB_GAS_NORMALIZATION_TARGET_BLOBS ` since it's `=1` and also all the terms are in the same unit, blob gas.

@g11tech wdyt?

Reply:
because (parent.blob_gas_used - target_blob_gas) // target_blob_gas) will be most likely evaluate to 0 int and we would want error accumulation to be some sensitive range which is what the above equation does